From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tim Pepper <timothy.c.pepper@linux.intel.com>
Date: Thu, 16 Feb 2017 13:54:50 -0800
Subject: [PATCH] golang stateless: fix /etc/services path in net

Forward port of prior patch from Dimitri John Ledkov in 2015.

Signed-off-by: Tim Pepper <timothy.c.pepper@linux.intel.com>
---
 src/net/port_unix.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/net/port_unix.go b/src/net/port_unix.go
index 4fdd9a3..5f2d7c9 100644
--- a/src/net/port_unix.go
+++ b/src/net/port_unix.go
@@ -16,7 +16,7 @@ import (
 var onceReadServices sync.Once
 
 func readServices() {
-	file, err := open("/etc/services")
+	file, err := open("/usr/share/defaults/etc/services")
 	if err != nil {
 		return
 	}
